OpenOversight/app/templates/officer.html

```diff
@@ -4,7 +4,7 @@
 <div class="container" role="main">
 
   <div class="page-header">
-    <h1>Officer Detail: <b>{% include 'partials/officer_name.html' %}</b></h1>
+      <h1><a href={{url_for('main.list_officer',department_id=officer.department.id)}}>{{ officer.department.name|title }}</a> >> <b>{% include 'partials/officer_name.html' %}</b></h1>
```

Comment:
Looks good! So one idea: there are styled breadcrumbs in bootstrap (which we use) - see [here](https://getbootstrap.com/docs/3.3/components/#breadcrumbs), want to try using those? They look pretty nice